src/runner.rs

```diff
@@ -221,10 +232,11 @@ impl Interpreter {
 		Ok(opt_return_value)
 	}
 
-	pub fn resume_execution<'a, E: Externals + 'a>(
+	pub(crate) fn resume_execution<'a, E: Externals + 'a>(
```

Comment:
As above: why is this crate-internal?

Reply:
Same reason as `start_execution`. It's not type safe.. yet.